activities/value-and-impact/pain-points/public-organizations.md

```diff
@@ -0,0 +1,34 @@
+---
+type: Resource
+---
+
+# Public organizations maturing in their approach to digital public infrastructure
+
+This resource contains an unordered and non-exhaustive list of painpoints that public organizations have shared with us during discussions about membership of the Foundation for Public Code and codebase stewardship.
+This resource can be used to inform our user research, value proposition mapping and business case development for potential members, as well as provide a general overview of the context in which we operate.
+
+These are common challenges faced by public organizations as they mature in their approaches to their digital public infrastructure.
+
+## Operational delivery
+
+Developing the exertise to operationally manage digital innovation projects, including in-house development, procurement, and the management of third party vendors, often without in-house technical expertise.
+
+## Economies of scale and silos
+
+Learning to work together with partner-organizations to collaborate around a single shared system, rather than each having their own.
```

Comment:
```suggestion
Learning to work together with partner organizations to collaborate around a single shared system, rather than each having their own.
```